common/lib/xmodule/xmodule/js/src/problem/edit.coffee

```diff
@@ -433,42 +433,86 @@ class @MarkdownEditingDescriptor extends XModule.Descriptor
           // Line split here, trim off leading xxx= in each function
           var answersList = p.split('\n'),
 
-              processNumericalResponse = function (value) {
-                  // Numeric case is just a plain leading = with a single answer
-                  value = value.replace(/^\=\s*/, '');
-                  var params, answer, string;
+              processNumericalResponse = function (values) {
+                  var values, params, answer, firstAnswer, string, additionalAnswerString;
 
-                  var textHint = extractHint(value);
+                  isRangeToleranceCase = function(answer) {
+                      if (_.contains([ '[', '(' ], answer[0]) && _.contains([ ']', ')' ], answer[answer.length-1]) ) {
+                        return true;
+                      }
+                      return false;
+                  }
+
+                  getAnswerData = function(answerValue) {
+                      var answerData = {};
+                      var answerParams = /(.*?)\+\-\s*(.*?$)/.exec(answerValue);
+                      if(answerParams) {
+                          answerData.answer = answerParams[1].replace(/\s+/g, ''); // support inputs like 5*2 +- 10
+                          answerData.default = answerParams[2];
+                      } else {
+                          answerData.answer = answerValue.replace(/\s+/g, ''); // support inputs like 5*2
+                      }
+                      return answerData;
+                  }
+
+                  // First string case is s?= [e.g. = 100]
+                  firstAnswer = values[0];
+                  firstAnswer = firstAnswer.replace(/^\=\s*/, '');
```

Comment:
Actually I am using `firstAnswer` below many times, so that's why I have saved this.